### ISSUE TITLE
Changes to the User-Agent

### ISSUE THREAD

**Issue by jurasjo** [UnclePhil/node-red-contrib-fr24tube#1]
https://github.com/derhuerst/flightradar24-client/issues/29

**Comment by tux2000** [comment]
Now there is a fix upstream for the User agent. But this is still not working locally for me, and i'm not sure if its something with my setup or not. @jurasjo, is this node working for you with the updates in flightradar24-client?

**Comment by jurasjo** [comment]
Yes, but I have done the change directly to the node modules files in my HA.
![Zrzut ekranu z 2023-07-08 20-04-45](https://github.com/UnclePhil/node-red-contrib-fr24tube/assets/19259405/9116ffc1-7eb1-469c-8d89-efa1595ab3bd)

**Comment by tux2000** [comment]
Oh, i see. For me the radar.js and flight.js are only within the node-red docker container. But i confirmed that they got the upstream updates from flightradar24-client. So will need to search more where the problem is on my end...

**Comment by tux2000** [comment]
Got it working yesterday after specifying the 1.0.0 dependency version and updating the import in fr24tube on a forked version of the code here: https://github.com/tux2000/node-red-contrib-fr24tube

Will make a PR soonish
